src/query/graphite/common/engine.go

```diff
@@ -54,3 +56,8 @@ func (e *Engine) FetchByQuery(
 ) (*storage.FetchResult, error) {
 	return e.storage.FetchByQuery(ctx, query, options)
 }
+
+// Storage returns the engine's storage object
```

Comment:
nit: by go-doc convention, the comment should go on the interface above, and it may need a `.` at the end

Reply:
Good to know! Fixed.